### PR TITLE
Handle token renewal redirect

### PR DESCRIPTION
**Pull request by eherot**
Giving this to @andrhamm because it's all rubycode

### CODE REVIEW

File: recipes/renew_vault_token.rb [hunk 1]
```diff
@@ -6,6 +6,27 @@
     begin
       require 'net/http'
 
+      def fetch(uri_str, vault_token, limit = 10)
+        # You should choose a better exception.
```
Comment (andrhamm) [inline recipes/renew_vault_token.rb]: what is this gray text, mate

**Comment by andrhamm** [comment]
http://i.imgur.com/9uklcyp.gif

**Comment by eherot** [comment]
@andrhamm gave his 👍

**Comment by eherot** [comment]
wat

**Pull request merged**